src/pages/home/Employee/EmployeeHome.css

```diff
@@ -0,0 +1,32 @@
+.logo {
```

Comment:
The CSS of the form is not matching with the figma design we should refactor a bit.